templates/envproxy.yaml

```diff
@@ -28,7 +28,9 @@ spec:
       restartPolicy: Always
       # 4 hours
       terminationGracePeriodSeconds: {{ .Values.envproxy.terminationGracePeriodSeconds }}
-      serviceAccountName: envproxy
+      # envproxy itself doesn't use this service account, but instead forwards
+      # the account token to the cemanager.
+      serviceAccountName: coder
```

Comment:
This means envproxy itself can also easily make kube calls. There could be a future where envproxy runs the controllers and sends build events back to cemanager. (this idea was in the OG resource pools rfc I think) to keep the kube data local and move the responsibility out of cemanager altogether.

Reply:
I had to do this because it's difficult to mount a second service account unless you know the secret's name exactly AFAIK.

Reply:
no I think this is better :+1: